meegkit/ress.py

```diff
@@ -29,92 +26,236 @@ def RESS(X, sfreq: int, peak_freq: float, neig_freq: float = 1,
         FWHM of the neighboring frequencies (default=1).
     n_keep : int
         Number of components to keep (default=1). -1 keeps all components.
-    return_maps : bool
```

Comment:
This should become a class attribute I think

Reply:
Then the name should change. In the original implem no matter `return_maps` the mixing and unmixing matrices are computed but returned only if `True`. 

In this implem both mixing and unmixing are also computed during the `fit` (`fit` returns `self`). Mixing matrix is mandatory for the transform so it has to be computed but unmixing matrix could be optional. 

Then I could add `compute_unmixing` attribute.